jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/resources/org/jbpm/workbench/cm/static/case-mgmt.less

```diff
@@ -258,5 +258,69 @@ dd:not(:last-child) {
   padding-left: 5px;
 }
 
+.kie-roles-item-line{
+  margin-left: 20px;
+  margin-top: -20px;
+  max-height: 21px;
+  overflow: hidden;
+  position : absolute;
+  width: 88%;
+  z-index: 1;
+}
+
+.kie-roles-item-line-expanded{
+  margin-left: 20px;
+  margin-top: -12px;
+  border: 1px solid #bbb;
+  background-color: @color-pf-white;
+  position : absolute;
+  width: 88%;
+  z-index: 2;
+}
+
+.kie-more-info{
+  text-decoration: none;
+  float: right;
+  margin-top: 4px;
```

Comment:
_margin-top_ could be removed, it would actually better align with user and group icons.